benchmarks/benchmark_experiment.py

```diff
@@ -85,11 +85,7 @@ def is_available(self, experiment_config):
         experiment_config["accelerator"] == "cuda" and
         not experiment_config["xla"]):
       return False
-    if experiment_config["dynamo"] == "openxla_eval" and not (
-        experiment_config["xla"] and experiment_config["test"] == "eval"):
-      return False
```

Comment:
I think I need to revert this deletion. `openxla_eval` is only supposed to work with `eval` test.